app/src/main/java/com/example/umlandowallet/data/remote/Access.kt

```diff
@@ -29,14 +29,7 @@ interface Access {
 
     companion object {
```

Comment:
I think you might be able to do without the `create()` method in the interface, and simply create an `AccessImpl` object directly at the call-site.

For example if you remove the companion object entirely, the one call to the AccessImpl is in `WalletScreen` and would become
```kotlin
onClick = {
    CoroutineScope(Dispatchers.IO).launch {
        // Access.create().sync()
        AccessImpl().sync()
    }
}
```

Reply:
Or you could create the object above and simply reuse it every time you sync instead of creating a new one every time.

```kotlin
@Composable
fun WalletScreen() {
    val accessImpl = AccessImpl()
// ...
onClick = {
    CoroutineScope(Dispatchers.IO).launch {
        accessImpl.sync()
    }
}
```

Reply:
Updated in https://github.com/ConorOkus/uMlando-wallet/pull/29/commits/c4460253dba1159a96eca648ddc3dd402693ff4d